pwa/src/templates/templateParts/cronjobsForm/CreateCronjobsFormTemplate.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from "react";
+import * as styles from "./CronjobsFormTemplate.module.css";
+import { useForm } from "react-hook-form";
+import APIContext from "../../../apiService/apiContext";
+import FormField, { FormFieldInput, FormFieldLabel } from "@gemeente-denhaag/form-field";
+import { Alert, Button, Heading1 } from "@gemeente-denhaag/components-react";
+import { useTranslation } from "react-i18next";
+import APIService from "../../../apiService/apiService";
+import { InputText, Textarea } from "@conduction/components";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faFloppyDisk } from "@fortawesome/free-solid-svg-icons";
+import { useQueryClient } from "react-query";
+import { useCronjob } from "../../../hooks/cronjob";
+
+interface CreateCronjobFormTemplateProps {
+  cronjobId?: string;
+}
+
+export const CreateCronjobFormTemplate: React.FC<CreateCronjobFormTemplateProps> = ({ cronjobId }) => {
+  const { t } = useTranslation();
+  const API: APIService | null = React.useContext(APIContext);
+  const [loading, setLoading] = React.useState<boolean>(true);
```

Comment:
Besides initiating it `true`, we never actually set it to anything else. We can do this, by adding the following effect:

```
React.useEffect(() => {
  setLoading(createOrEditCronjob.isLoading);
}, [createOrEditCronjob]);
```

**Note: this only works when we remove the Throw on errors within the useCronjob hook.**

So instead of:

```
onError: (error) => {
  throw new Error(error.message);
},
```

We do: 

```
onError: (error) => {
  console.error(error.message);
},
```